src/init.py

```diff
@@ -0,0 +1,157 @@
+from os import getenv
+import shutil
+import subprocess
+from pathlib import Path
+from itertools import product
+import build
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+import logging
+import sys
+
+
+def getvar(var: str) -> str:
+    val = getenv(var)
+    if val == "" or val is None:
+        raise ValueError('Environment variable "%s" has an invalid value.' % var)
+    return val
+
+
+def make_key(key_path: str, key_subj: str) -> None:
+    subprocess.run(
+        ["/root/make_key", key_path, key_subj],
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+        check=True,
+        input="\n".encode(),
+    )
+
+
+class Init:
+    def __init__(self) -> None:
```

Comment:
Why a class? The script runs exactly once, from top to bottom. I don't see the point in having all those properties initialized in the constructor when you only use them once later, it makes the code harder to understand.

Reply:
The benefit I saw was that the test could access the keys it needs to check for from the init object, rather than needing to maintain a list separately.  But I'm happy to revert it back if you want.